dhall/src/Dhall/Pretty/Internal.hs

```diff
@@ -199,8 +204,13 @@ renderSrc strip (Just (Src {..}))
                 in  Pretty.pretty l0 : map perLine ls
 
     f x y = x <> Pretty.hardline <> y
-renderSrc _ _ =
-    mempty
+
+prettyComment :: Maybe Src -> Maybe (Doc Ann)
```

Comment:
I'm not very happy with this name and the `renderSrc` and `renderComment` names above yet.

Reply:
Would this be better?!

```suggestion
prettySrc :: Maybe Src -> Maybe (Doc Ann)
```

Reply:
I renamed `prettyComment` to `renderSrcMaybe`.